docs/UserGuide.md

```diff
@@ -90,12 +90,12 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a person to the contact list.
```

Comment:
for everyone, I think we should standardize and call it a contact list. Some of us still refer to it as addressbook.

@evanpy @wpx12011 @ajjajjajjajj @benedict733